changelogs/unreleased/gh-9806-http-set-version.md

```diff
@@ -0,0 +1,3 @@
+## feature/lua/http client
```

Comment:
Please, add a subsystem prefix into the commit message subject. For example, `httpc:`. The same for the pull request title.

Reply:
Fixed

Reply:
And the pull request title, please.

Reply:
Fixed